src/main/resources/data.sql

```diff
@@ -25,14 +26,15 @@ create table payment (
 );
 
 create table profile (
-    id varchar(60) primary key,
+    profile_id varchar(60) primary key,
```

Comment:
Profile id was changed from id and the others were not.  Naming is an interesting beast.  In this case, with the others having only id as the field, changing only this one may cause confusion.  https://thecoderoad.blog/2020/03/29/clean-code-naming-conventions/

Reply:
I see. I should have changed to other to match (userId, addressId, etc) to follow convention.

Thanks for the link. I'll read up on it